vignettes/Update-from-openxlsx.Rmd

```diff
@@ -120,7 +122,7 @@ wb <- wb_workbook("My name here")$
   )$
   set_col_widths( # set column width
     cols = 1:6,
-    widths = c("20", rep("10", 5))
+    widths = c(20, rep(10, 5))
```

Comment:
Removing quotes, as these are integers?

Reply:
Yes, but likely from the old days, when we did not support internal conversion from integer/numeric to `as_xml_attr()`.